src/listeners/ready.ts

```diff
@@ -24,16 +24,10 @@ export class ReadyListener extends Listener {
     this.printBanner();
     this.printStoreDebugInformation();
     this.sendReady();
-    this.initCrons();
+    initCrons();
```

Comment:
curious why this this this then no more this

Reply:
Local methods vs methods imported from other files.
I moved init crons to the cron file because frankly that makes more sense